JarrydWorland/2021_Capstone_GP2_Team
# Fixed door getting stuck in expanded state

## Description
When standing next to a door and clearing a room, the door expanding caused the player to get sent back to the previous room. This also caused the door to be stuck in its expanded state preventing the player from re-entering the room, as when they entered the room they would be moved onto the collider and get immediately sent back.

## Changes include
- Added idle check to DoorConnectionBehaviour IsOpen (57166c0)

## How to test
Enter a room with enemies, and stand up close against a door while clearing the room. When the door opens the player should not be immediately sent back to the previous room, the doors should at no point be frozen in an expanded state.
